src/code.cloudfoundry.org/cf-tcp-router/config/config_test.go

```diff
@@ -272,7 +272,7 @@ var _ = Describe("Config", Serial, func() {
 			})
 		})
 
-		Context("with invalid cert and key", func() {
+		Context("with invalid cert and key and enableCertCreation set to True", func() {
```

Comment:
This context SAYS it is about when there is an invalid cert and key, but a few lies down it says it is valid and everything works.

Reply:
true, let me add some more coverage for the error cases as well